example/scripts/run_test.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-set -e
+set +e
```

Comment:
There was a bug in the test output where it wouldn't output the exit code if the test failed so this is so we can see failures in travis earlier.